cmd/keys/export/export.go

```diff
@@ -40,10 +41,13 @@ func Cmd() *cobra.Command {
 				out, err := utils.ExecBashCommand(exportKeyCmd)
 				printKeyOutput(out.String(), err)
 			} else if keyID != "" && keyID == damanager.GetKeyName() {
+				//TODO: avail doesn't need cmd to get the keys, it's stored in the config
```

Comment:
avail doesn't get included in the supported keys in this flow right

Reply:
I know, there should be issue to handle it